### PR TITLE
Fix: `WsEvent::Closed` is correctly sent on web when socket is closed.

### PR DESCRIPTION
**Pull request by vincev**
I have noticed that when a Websocket server closes the connection the client wasn't getting a close notification, this PR adds a close callback so that the event is propagated to the UI:

CloseNotify

### CODE REVIEW

**Review by emilk — APPROVED** [review]
Great - thanks!

**Comment by emilk** [comment]
I'm updating to 1.65 to fix the CI issues: https://github.com/rerun-io/ewebsock/pull/7

**Comment by emilk** [comment]
Try merging in `main`

**Comment by emilk** [comment]
Please add a line to `CHANGELOG.md` btw!

**Comment by vincev** [comment]
> Please add a line to `CHANGELOG.md` btw!

I have updated CHANGELOG.md and the version, I wasn't sure if I needed to add that under `Unreleased`, I can update it if that is your preference.

**Review by emilk — APPROVED** [review]

**Comment by emilk** [comment]
It should go under `Unreleased`, because it is unreleased :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ## Unreleased
+* Add callback for close notification to WASM WebSocket.
```
Comment (emilk) [inline CHANGELOG.md:2]:
```suggestion
* Fix: `WsEvent::Closed` is correctly sent on web when socket is closed.
```

File: ewebsock/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ewebsock"
-version = "0.2.0"
+version = "0.2.1"
```
Comment (emilk) [inline ewebsock/Cargo.toml:3]:
```suggestion
version = "0.2.0"
```

**Pull request merged**